src/renderer/scss/component/_card.scss

```diff
@@ -2,321 +2,257 @@
   margin-left: auto;
```

Comment:
ignore this for now, it will change a lot

I've commented out a lot to see what we actually need. Will be better suited for a review after I work on the file page